src/components/videocard/VideoCard.css

```diff
@@ -0,0 +1,103 @@
+.vertical-card {
+  min-width: 320px;
+  height: fit-content;
+  overflow: hidden;
+  background-color: var(--card-bgc);
+  margin: 10px 0px;
+  padding: 0 0 5px 0;
+}
+.vertical-card:hover {
+  transform: scale(1.05);
+  transition: 0.5s ease-in-out;
+  z-index: 9999;
+}
+.img-container {
+  position: relative;
+  width: 320px;
+  min-height: 200px;
+  overflow: hidden;
+  z-index: 1;
+}
+
+.play-btn {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 9999999;
```

Comment:
You see here the number is even bigger. so either make variables with level or have some pre-defined number so it does not create chaos at a later point